src/misc/interfaces/IERC6909.sol

```diff
@@ -116,6 +116,12 @@ interface IERC6909TotalSupplyExt {
     function totalSupply(uint256 tokenId) external returns (uint256 supply);
 }
 
+interface IERC6909Decimals {
```

Comment:
Should `IERC6909MetadataExt` extends from `IERC6909Decimals`?

Reply:
Yeah I was still not sure who to set up these interfaces now. That seems like the simplest solution but then the interface ID of `IERC6909MetadataExt` will be wrong, since interface IDs don't account for inherited interfaces.

Reply:
😲 interesting!